webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -28,4 +48,146 @@ def calculate_pauschbetrag(has_pflegegrad=False, disability_degree=None, has_mer
     elif disability_degree >= 30:
         return 620
     elif disability_degree >= 20:
-        return 384
\ No newline at end of file
+        return 384
+    
+    return 0
+    
+    
+class PersonAHasDisabilityPrecondition(DisabilityModel):
+    _step_to_redirect_to = StepPersonAHasDisability.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_disability')
+
+    @root_validator(skip_on_failure=True)
+    def person_b_has_disability(cls, values):
+        if values.get('person_a_has_disability') != 'yes':
+            raise ValidationError
+        return values
```

Comment:
Why are you making this a root_validator? I think this could just be a normal validator of one field - as it only checks one single field:
```suggestion
    @alidator('person_a_has_disability', always=True)
    def must_not_be_yes(cls, v):
        if v != 'yes':
            raise ValidationError
        return values
```

Reply:
Thanks! :grin: did not know there is a normal validator